src/classBased/components/TodoItem.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable react/button-has-type */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/prop-types */
+/* eslint-disable indent */
+/* eslint-disable react/sort-comp */
+/* eslint-disable semi */
+/* eslint-disable react/state-in-constructor */
```

Comment:
[OPTIONAL]
- Avoid deactivating too many linters, this may lead to a weak code structure. Don't worry about it now, but keep it in mind for the next projects.